phpstan-baseline-deprecated.neon

```diff
@@ -1501,15 +1501,15 @@ parameters:
 				#^Call to deprecated function dbFetchCell\\(\\)\\:
 				Please use Eloquent instead; https\\://laravel\\.com/docs/eloquent$#
 			"""
-			count: 6
+			count: 10
```

Comment:
Hello,
This is not the goal to increase these values :D Of course, lowering them is allowed :) you can mix eloquent requests in an existing file (and keep count==6, or even better, change some DbFacile requests to eloquent and lower this value)